hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -91,7 +92,8 @@ public class AbfsClient implements Closeable {
   private final URL baseUrl;
   private final SharedKeyCredentials sharedKeyCredentials;
   private final String xMsVersion = "2019-12-12";
-  private final ExponentialRetryPolicy retryPolicy;
+  private final ExponentialRetryPolicy exponentialRetryPolicy;
```

Comment:
i think these should both be of the type of the baseclass, and rather than name exponential static, be `apiRetryPolicy` and `connectivityRetryPolicy`. that way the policy can be changed without break all field/accessor method names.

Reply:
At this moment, in AbfsClient these refer the nature of retry policy and not the context in which they will be used.
In future we might end up using some other retry policy for some other kind of failure.

I think it makes sense to keep these names for now.
We can think on above lines when we take up refactoring of whole retry policy infra in ABFS.